cloudify_agent/shell/commands/daemons.py

```diff
@@ -382,3 +418,35 @@ def _parse_custom_options(options):
         parsed[key] = value
 
     return parsed
+
+
+def _create_rest_ssl_cert(agent):
+    """
+    put the REST SSL cert into a file for clients to use,
+    if rest_cert_content is set.
+    """
+    click.echo('Deploying REST SSL certificate (if defined).')
+    rest_cert_content = agent['rest_cert_content']
+    if rest_cert_content and agent['verify_rest_certificate']:
+        local_rest_cert_file = os.path.join(agent['workdir'], 'rest.crt')
+        agent['local_rest_cert_file'] = local_rest_cert_file
+        api_utils.safe_create_dir(os.path.dirname(local_rest_cert_file))
+        with open(local_rest_cert_file, 'w') as rest_cert_file:
+            rest_cert_content = rest_cert_content.replace("\\n", "\n")
```

Comment:
do we get the value here as json? if so, can't you simply do json.loads?
also, considering the certificate is a simple base64 one, i honestly don't think the extra utf-8 encoding is required, as it is all ascii anyway.